integration-tests/e2e_tests_ci_tool/cmd/types.go

```diff
@@ -0,0 +1,25 @@
+package cmd
+
+type Test struct {
+	Name string
+	Path string
+}
+
+// CITestConf defines the configuration for running a test in a CI environment, specifying details like test ID, path, type, runner settings, command, and associated workflows.
+type CITestConf struct {
+	ID                    string            `yaml:"id" json:"id"`
+	IDSanitized           string            `json:"idSanitized"`
```

Comment:
Nitpick, do we need both `yaml` and `json` tags?

Reply:
@skudasov `yaml` is used for [.github/e2e-tests.yml](https://github.com/smartcontractkit/chainlink/blob/TT-1297-Create-nightly-pipeline-for-all-e2e-tests-in-core/.github/e2e-tests.yml). And `json` is used for `test-inputs` map in `CITestConf` because it's way easier to handle it this way in github workflows. On the other hand we still want to use yaml for .github/e2e-tests.yml because yaml is much easier to read, allows comments, etc. ([yaml vs json](https://aws.amazon.com/compare/the-difference-between-yaml-and-json/))